src/components/search/History/SearchHistory.tsx

```diff
@@ -20,36 +21,33 @@ const doSearch = (term: string) => {
   return `${pathname}${search}`
 }
 
-const SearchHistory = ({ onClear }: SearchHistoryProps) => {
-  const { searchHistory } = useSearchHistory()
+const SearchHistory = ({ history = [], onClear }: SearchHistoryProps) => {
+  const { searchHistory } = useSearchHistory(history)
 
   return (
-    <section data-store-search-history>
-      <div data-store-search-history-header>
-        <h4 data-store-search-history-title>History</h4>
+    <section data-fs-search-suggestion-section>
+      <div data-fs-search-suggestion-header>
+        <p data-fs-search-suggestion-title>History</p>
         <Button variant="tertiary" onClick={onClear}>
           Clear
         </Button>
       </div>
       <UIList variant="ordered">
         {searchHistory.map((item, index) => (
-          <li data-store-search-history-item key={index}>
+          <li key={index} data-fs-search-suggestion-item>
```

Comment:
What do you think about using another key, instead of the index here? In this case, I think the key must be unique (maybe an ID or something like that). 

[This article](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318) explains that a little bit.

Reply:
Thanks for the article! I changed it [here](https://github.com/vtex-sites/gatsby.store/pull/7/commits/544924fcdf912f49199265a308e06ccc8fe59c36).